components/server/ee/src/user/stripe-service.ts

```diff
@@ -154,15 +154,12 @@ export class StripeService {
                 taxInformation: customer.tax,
             });
         }
-        const startOfNextMonth = new Date(new Date().toISOString().slice(0, 7) + "-01"); // First day of this month (YYYY-MM-01)
-        startOfNextMonth.setMonth(startOfNextMonth.getMonth() + 1); // Add one month
 
         await reportStripeOutcome("subscriptions_create", () => {
             return this.getStripe().subscriptions.create({
                 customer: customer.id,
                 items: [{ price: priceId }],
                 automatic_tax: { enabled: isAutomaticTaxSupported },
-                billing_cycle_anchor: Math.round(startOfNextMonth.getTime() / 1000),
```

Comment:
Doesn't the corresponding change need to be in `CreateSubscription`, instead of `CreateStripeCustomer`? In this case, I believe I've already adjusted the two corresponding code locations.

Reply:
Sorry, yes - in CreateSubscription

Reply:
Then this is already resolved 🙂 ✅

Reply:
Did this happen in this PR? I would've expected the same PR changing the logic in both branches of the feature flag.

Would you like to create a pre-PR which removes this method (and removes the experiment) such that this change is not needed in the first place?

Reply:
> Did this happen in this PR?

Yes: Please look at the diff, you can search for "BillingCycleAnchor"

Reply:
Also, note that it might be easier to review each commit separately -- there you'll see that both branches were in fact changed by the same logical commit.

Reply:
Thanks.